### DISCUSSION TITLE
Sorry for lack of maintenance

### DISCUSSION THREAD

**Discussion by djmaze** [djmaze/resticker#129] [General]
Had some time and motivation issues. Will try to get this project up-to-date again. 

Regardless, a secondary maintainer would be very welcome!

**Comment by bread-wolf** [comment]
No worries, I figured out the use of the variable I was looking for, the container has been running for 3 months without interruption now.
I could provide examples for docker-compose files and a tutorial to set up the container with rclone and google drive if you want. 

As for maintaining, I haven't looked at how this all works in more than three months but I would be happy to help where I can.

**Reply by djmaze** [reply]
I guess more examples would be helpful. We should put them into a subfolder probably.

I will cut a new release next week. If others would like to help, I could mark suitable issues as "help wanted".

**Comment by ThomDietrich** [comment]
@djmaze I hear you. It is a challenge to keep your own repos active. I had similar challenges over the years and even when I started to become more active with your project here, I ended up deprioritizing my contributions. I kind of remember that I had some enthusiastic ideas for further PRs but never followed up on those, sorry about that!

Anyhow, what could make the concept of secondary maintainer(s) work: Collect a number of friends of the project in a team (https://docs.github.com/en/organizations/organizing-members-into-teams/about-teams). Then whenever an issue, pull request, or feature discussion could benefit from input by these people, you can mention the team handle. Works quite well in other projects.

I would be honored to be part of a group of secondary maintainers.

**Reply by djmaze** [reply]
@ThomDietrich Thanks for your thoughts and feedback. No need to be sorry, I know how it works. You have times when a project is at the center of your attention (when you are actively pursuing to satisfy your own immediate requirements) and then there are even many other things to focus on. Also, it depends on the activeness and response time of the maintainer, which I admit was far from perfect in the past.

Your idea about "secondary maintainers" sounds interesting. Can you point me to projects where this is being successfully used, as an example?

**Reply by ThomDietrich** [reply]
Sure, I saw this in other projects as well but one where it made a lot of sense to me is in the multiple repos hosted under openHAB. A few teams are defined: https://github.com/orgs/openhab/teams
And these are then referenced where it makes sense, e.g. https://github.com/openhab/openhab-core/issues/2283#issuecomment-922250700=

**Reply by djmaze** [reply]
Mhh, I am immediately redirected away from the teams page. I also cannot see any team reference in the linked comment. Guess that's because I am not a member of the organization.

**Reply by ThomDietrich** [reply]
![image](https://user-images.githubusercontent.com/2870104/169767965-cab78a02-fd24-4e28-a0eb-8034142e8d7f.png)

->

![image](https://user-images.githubusercontent.com/2870104/169768083-af5afa60-80fb-47a5-a5de-69f35445e3bf.png)
